### PR TITLE
Add OpenAPI chart

### PR DESCRIPTION
**Pull request by makoscafee**
**Why This PR?**
_a short description of why this PR is needed_

Fixes #

**Changes**
Changes proposed in this pull request:

*
*

**Checklist:**

* [ ] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [ ] Any new values are backwards compatible and/or have sensible default.
* [ ] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

**Review by rbren — APPROVED** [review]

File: stable/fairwinds-insights/templates/ingress.yaml [hunk 1]
```diff
@@ -122,6 +126,10 @@ spec:
             {{ else }}
             path: /
             {{ end }}
+          - backend:
+              serviceName: {{ $fullName }}-open-api
+              servicePort: http
+            path: /swagger
```
Comment (rbren) [inline stable/fairwinds-insights/templates/ingress.yaml:139]: Just need to support `starPaths` like above

**Comment by makoscafee** [comment]
so, swagger update `usr/share/nginx/html/index.html` with the `SWAGGER_JSON_URL` just after the container has been  created. i have changed `readOnlyRootFilesystem` to default `false` not sure this is the best approach though.

File: stable/fairwinds-insights/templates/deployment-open-api.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        - name: fairwinds-insights
+          image: "{{ .Values.openApiImage.repository }}:{{ include "fairwinds-insights.openApiImageTag" . }}"
+          imagePullPolicy: Always
+          ports:
+          - name: http
+            containerPort: {{ .Values.openApi.port }}
+            protocol: TCP
+          env:
+          - name: SWAGGER_JSON_URL
+            value: {{ include "fairwinds-insights.fullname" . }}-api/v0/swagger.json
```
Comment (rbren) [inline stable/fairwinds-insights/templates/deployment-open-api.yaml:34]: does this work? Just the service name?

**Comment by rbren** [comment]
Hmm - I don't see another way around it. Thanks Barnie!

**Pull request merged**